### DISCUSSION TITLE
Welcome to AdultMediaManager Discussions!

### DISCUSSION THREAD

**Discussion by adultmm** [adultmm/AdultMediaManager#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Can't find file

**Comment by ghost** [comment]
How do you download it on to kodi

**Comment by adultmm** [comment]
Hi @Aaronhmail1994,

It's not a Kodi plugin. It's a Windows application to organize your movie collection then Kodi'll load the meta info and posters of the movies.

**Comment by justaguyinaz** [comment]
How do you tell it where to find your media files? Once it finds data, does it automatically add it to the media file itself, or does it all just get embedded in a db?

**Comment by Ahaezio** [comment]
这个程序对新手很不友好！无法找到加载影片的路径，更别提怎么刮削元数据。我需要使用方法。

**Comment by thelion55** [comment]
it's possible you can make it for mac users?
